engine/src/main/java/io/zeebe/engine/state/message/DbMessageSubscriptionState.java

```diff
@@ -88,12 +90,13 @@ public MessageSubscription get(final long elementInstanceKey, final DirectBuffer
   }
 
   @Override
-  public void put(final MessageSubscription subscription) {
-    elementInstanceKey.wrapLong(subscription.getElementInstanceKey());
-    messageName.wrapBuffer(subscription.getMessageName());
-    subscriptionColumnFamily.put(elementKeyAndMessageName, subscription);
+  public void put(final long key, final MessageSubscriptionRecord record) {
+    elementInstanceKey.wrapLong(record.getElementInstanceKey());
+    messageName.wrapBuffer(record.getMessageNameBuffer());
+    messageSubscription.setKey(key).setRecord(record).setCommandSentTime(0);
```

Comment:
Can we write it like 
```suggestion
    messageSubscription
      .setKey(key)
      .setRecord(record)
      .setCommandSentTime(0);
```

It was hard for me to detect that we setting these properties

Reply:
or add some new lines